src/util.js

```diff
@@ -0,0 +1,40 @@
+// @flow strict
+import { EVERGREEN_BASE } from "./config";
+
+const getTestMetadataURL = ({
+  taskId,
+  execution,
+  testId,
+}: {
+  taskId: string,
+  execution: number,
+  testId: string,
+}) =>
+  `${EVERGREEN_BASE}/rest/v2/tasks/${taskId}/tests?execution=${execution}&test_id=${testId}`;
+
+export const getTestMetadata = async ({
+  taskId,
+  testId,
+  execution,
+}: {
+  taskId: string,
+  testId: string,
+  execution: number,
+}) => {
+  const testMetadataUrl = getTestMetadataURL({ execution, taskId, testId });
+  const testMetadataReq = new Request(testMetadataUrl);
+  const res = await window.fetch(testMetadataReq, {
+    credentials: "include",
+  });
+  const data = await res.json();
+  if (Array.isArray(data) && data.length) {
+    if (data.length > 1) {
+      console.error(
+        "Multiple results error: multiple results came back from the rest v2 tests endpoint instead of 1. Url:",
+        testMetadataUrl
+      );
+    }
```

Comment:
When would this happen?

Reply:
this condition prevented us from moving forward in the past and is something i would log to bugsnag in case it affects our users again. Also, the api endpoint always returns an array and we could avoid this check in the frontend if it were an obj instead of an arr